decidim-core/spec/tasks/decidim_tasks_right_to_be_forgotten_spec.rb

```diff
@@ -35,21 +35,22 @@
       task.execute
       check_no_errors_have_been_printed
       expect(Decidim::User.where(id: user_ids).all?(&:deleted?)).to be true
-      check_message_printed("[#{user_ids.first}] DELETING USER")
+
+      expect($stdout.string).to include("[#{user_ids.first}] DELETING USER")
     end
 
     it "ignores not found users" do
       user_id = user
       create_forgotten_users_file([user_id, 123_456])
       task.execute
-      check_message_printed("[123456] User not found")
+      expect { check_message_printed("[123456] User not found") }.not_to raise_error
```

Comment:
instead of doing a double expect here, I'd prefer to just inline (here and in other places) the `check_message_printed` method, so it is always the pattern as in line 39. On this case this would be like 

```suggestion
      expect($stdout.string).to include("[123456] User not found")
```

Reply:
I'd do this on all the cases where we're calling to check_message_printed so we can fully remove that method

Reply:
fixed by [817b795](https://github.com/decidim/decidim/pull/14508/commits/817b7951b7bfcb78e1077477d00efc91d32f42f1)